### DISCUSSION TITLE
How many years per generation?

### DISCUSSION THREAD

**Discussion by clawsoon** [tskit-dev/tsdate#196] [Q&A] [answered]
My run of the unified_genealogy_paper repo for chromosome 22 has generated an oldest time of 81,721, and times for the ancients ranging from 184 to 4,400.  I'm guessing those are generations.

How many years per generation should I use?

For some reason I have 28 years in my head from somewhere, and it seems to give reasonable-ish(?) results for the ancients (56kya for Vindija, 72kya for Denisova, 90kya for Chagyrskaya-Phalanx, 123kya for Altai, 5kya for Afanasievo), but I can't find the reference now.

Thanks.

**Comment by clawsoon** [comment, marked as answer]
I believe that I've found my answer, and it is indeed 28:

https://github.com/awohns/unified_genealogy_paper/blob/master/src/constants.py

**Reply by awohns** [reply]
Hi @clawsoon, thanks for your question. I've actually just updated the repo, please use 25 as the generation time for this data.

**Reply by clawsoon** [reply]
Thanks, @awohns.
